protractor-docker/docker-selenium-grid-conf.js

```diff
@@ -0,0 +1,19 @@
+exports.config = {
+	framework: 'jasmine2'
+	, specs: ['specs/*.js']
```

Comment:
Could we have a spec file that does a really simple test? It could just make a request for angularjs.org and check to see if the title matches.